charts/images.yaml

```diff
@@ -64,7 +64,7 @@ images:
 - name: csi-driver
   sourceRepository: github.com/kubernetes-sigs/aws-ebs-csi-driver
   repository: public.ecr.aws/ebs-csi-driver/aws-ebs-csi-driver
-  tag: "v1.9.0"
+  tag: "v1.11.2"
```

Comment:
In https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1333 the securityContext in the upstream chart is improved. Can you reflect that changes to our charts?

Reply:
@ialidzhikov  Can you PTAL ?